README.md

```diff
@@ -194,6 +194,13 @@ Adding the `toggle-btn` class to your radio inputs converts them to toggle butto
 <input type="radio" class="sbtn toggle-btn" />
 ```
 
+## Rounded buttons Radius
+To add the rounded buttons with a particular radius, you can use various classes like `rounded-x-btn` where x is the radius in pixels which you want. Here x ranges from `5`(border-radius = 5px) to `full`(border-radius = 50%).
+
+```html
+<button class="rounded-5-btn">Rounded button </button>
```

Comment:
add one of the base classes to style the button, for example `sbtn blue-btn`

Reply:
@shahednasser  I have done all the changes which you have told. Please review them.